bundle.Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=service-telemetry-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=latest
+LABEL operators.operatorframework.io.bundle.channels.v1=stable,latest
```

Comment:
Should probably revert this back to just `latest` since that's what the HEAD bundle should probably target.